docs/installation.md

```diff
@@ -46,9 +64,19 @@ Alternative you can clone the repository and do an editable install with:
 pip install -e .
 ```
 
-This is the reccommended way to install the package in case in which you want to contribute to the code. To update the package you can use `git pull`.
+This is the reccommended way to install the package in case in which you want to contribute to the code. In this case, to subsequently update the package you can use `git pull`.
```

Comment:
In that case, wouldn't we prefer people to create a fork first?

Reply:
```suggestion
An editable installation is the best option if you want to contribute to the development of spatialdata.
Check out our [contributing guide](https://spatialdata.scverse.org/en/latest/contributing.html).
```

explaining forking goes beyond the scope of installation instructions.
But we can use the opportunity to funnel people to the development guide.

Reply:
Yes in any case for Numfocus we do require a contributors guide.

Reply:
Thanks for the comments. It's hidden but it's there: https://github.com/scverse/spatialdata/blob/main/docs/contributing.md.

I will modify installation.md to quickly mentione forks.